components/event-bus/cmd/subscription-controller/cmd/manager/main.go

```diff
@@ -72,6 +85,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	log.Info("Setting up Knative Channel Controller")
+	_, err = knativechannel.ProvideController(mgr)
```

Comment:
Does it make sense to change the function signature to return only an error? since it is ignored.

Reply:
Right, we don't use the controller obj anyway. Fixed.